### ISSUE TITLE
WIP for eOS 6

### ISSUE THREAD

**Issue by Zonnev** [Zonnev/elementaryos-firefox-theme#85]
Hello, I installed eOS 6 in a VM and got busy restyling the tabs to the new standard. Below you can see how it looks like right now. I also provided a link to download the new userChrome with the newly styled tabs. Since eOS 6 is not completed yet, I will wait till release to complete the final styling of this theme. Till then, enjoy this version!
![Schermafdruk van 2020-09-07 19 11 08](https://user-images.githubusercontent.com/32688765/92408804-dca09800-f13e-11ea-803b-b6b8b7904c61.png)
![Schermafdruk van 2020-09-07 19 11 29](https://user-images.githubusercontent.com/32688765/92408820-e3c7a600-f13e-11ea-8f9f-d52d8a438545.png)
![Schermafdruk van 2020-09-07 19 10 37](https://user-images.githubusercontent.com/32688765/92408788-d6122080-f13e-11ea-9dba-3558343c6b41.png)
[userChrome.css.zip](https://github.com/Zonnev/elementaryos-firefox-theme/files/5184226/userChrome.css.zip)

**Comment by sempasha** [comment]
It looks like new styled tabs requires more vertical space, so it could be less convenient for small screen laptop owners.
May be it would be nice to let people to choose style they prefer - eOS5 or eOS6?

**Comment by Zonnev** [comment]
@sempasha Yes the tab bar height is a few pixels taller in the new eOS 6 style but it's consistent with the new standard. 

Now I'm thinking to keep the eOS 5 userChrome's in a second branch and eOS 6 styled userChrome's in the master branch. But in case of the install scripts - I don't know if it will work on the second branch, just by adjusting the adress to the script or if there needs to be more to be adjusted in the script itself. Or maybe you have a better idea how to keep both styles available on Github?

**Comment by Zonnev** [comment]
I see its not possible to make a new standalone branch in my repository, so i will just make a new folder and clone the eOS 5 files in there, when eOS 6 is ready.

**Comment by Zonnev** [comment]
I did it differently: I've duplicated this 'old' repository to a new repository: 
https://github.com/Zonnev/elementaryos5-firefox-theme

For now, if some changes will occure to the elementary OS 5 (Hera) theme, the changes will been seen in both old and new repositories. 

When elementary OS 6 (Odin) has arrived, this repository (old repository) will be updated with the new styles from new userChrome's. Also this way I won't lose the precious stars for this theme ;)

I hope all is clear now about what happens when eOS 6 arrives!

**Comment by Zonnev** [comment]
elementaryOS 6 (Odin) uses new back and forward buttons so I changed them as well. I also restyled private mode with the new tab bar background. Since private mode needed a tab bar background with a gradient from the top, it's not possible to have the background in a gradient colour like it used to be. So private mode has only one colour now (see the screenshots). Again there is an updated userChrome attached to this post. 

**Note:** elementaryOS 6 draws no border around the Firefox window, thats why the first screenshot looks a bit odd. I hope the devs will fix this matter in the final stages.

![Schermafdruk van 2020-10-14 23 55 45](https://user-images.githubusercontent.com/32688765/96050026-41a38800-0e79-11eb-87fd-b8fa2ace6b22.png)
![Schermafdruk van 2020-10-14 23 56 08](https://user-images.githubusercontent.com/32688765/96050037-449e7880-0e79-11eb-95a5-cc3ab2fa9713.png)
![Schermafdruk van 2020-10-14 23 56 52](https://user-images.githubusercontent.com/32688765/96050042-4700d280-0e79-11eb-906b-af7a202cf27b.png)
[userChrome.css.zip](https://github.com/Zonnev/elementaryos-firefox-theme/files/5381050/userChrome.css.zip)
